.jenkins/caffe2/test.sh

```diff
@@ -86,13 +86,22 @@ if [[ "$BUILD_ENVIRONMENT" == *-cuda* ]]; then
   EXTRA_TESTS+=("$caffe2_pypath/contrib/nccl")
 fi
 
-rocm_ignore_test=()
+ignore_tests=()
 if [[ $BUILD_ENVIRONMENT == *-rocm* ]]; then
   # Currently these tests are failing on ROCM platform:
 
   # On ROCm, RCCL (distributed) development isn't complete.
   # https://github.com/ROCmSoftwarePlatform/rccl
-  rocm_ignore_test+=("--ignore $caffe2_pypath/python/data_parallel_model_test.py")
+  ignore_tests+=( --ignore $caffe2_pypath/python/data_parallel_model_test.py )
+fi
+
+##############################
+# TensorRT integration tests #
+##############################
+if [[ $BUILD_ENVIRONMENT == *linux-xenial-cuda9-cudnn7-py3* ]]; then
+  pip install -q --user torchvision
+else
+  ignore_tests+=( --ignore $caffe2_pypath/python/trt/test_pt_onnx_trt.py )
```

Comment:
can we still keep running test_pt_onnx_trt.py?

Instead, if some tests are really slow, feel free to skip them by adding annotation @skipIf before the tests.

Reply:
@houseroad Yes, the idea is to run it but we can't run it for every environment. In TRT5 days we've chosen one env but it's gone now. So I'm trying others